docs/en/traditional-media.md

```diff
@@ -281,6 +281,30 @@ Support all channels, refer to [CNBC RSS feeds](https://www.cnbc.com/rss-feeds/)
 
 </RouteEn>
 
+## La Jornada
+
+### News
+
+<RouteEn author="thealf154" example="/lajornada/2022-10-12/capital" path="/lajornada/:date/:category?" :paramsDesc="['Date, use today to get String must be in format: `YYYY:MM:DD`. You can get today\'s news using `today` ', 'Category, refer to the table below']" radar="1">
```

Comment:
```suggestion
<RouteEn author="thealf154" example="/jornada/2022-10-12/capital" path="/jornada/:date/:category?" :paramsDesc="['Date string, must be in format of `YYYY-MM-DD`. You can get today\'s news using `today`', 'Category, refer to the table below']" radar="1">
```
- `YYYY-MM-DD` according to the example
- `/jornada/*` is a better choice as it matches the domain name.